GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
-                    <property name="can_focus">True</property>
-                    <property name="resize_mode">immediate</property>
-                    <property name="wrap_mode">word</property>
-                    <property name="left_margin">5</property>
-                    <property name="right_margin">5</property>
-                  </object>
-                </child>
+                <property name="icon-name">open-menu-symbolic</property>
+                <property name="icon-size">1</property>
               </object>
-              <packing>
-                <property name="expand">True</property>
-                <property name="fill">True</property>
-                <property name="position">0</property>
-              </packing>
             </child>
           </object>
           <packing>
-            <property name="expand">True</property>
-            <property name="fill">True</property>
-            <property name="position">1</property>
-          </packing>
-        </child>
-        <child>
-          <object class="GtkSeparator" id="hseparator1">
-            <property name="visible">True</property>
-            <property name="can_focus">False</property>
-          </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">False</property>
-            <property name="position">2</property>
+            <property name="pack_type">end</property>
           </packing>
+          <style>
+            <class name="image-button"/>
+          </style>
         </child>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox" id="vbox4">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
         <child>
-          <object class="GtkBox" id="tasksidebar">
+          <object class="GtkBox">
+            <property name="margin">3</property>
             <property name="visible">True</property>
-            <property name="can_focus">False</property>
-            <child>
-              <object class="GtkLabel" id="label2">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <property name="xalign">0</property>
-                <property name="label" translatable="yes">Starts on</property>
-                <attributes>
-                  <attribute name="weight" value="bold"/>
-                </attributes>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="padding">10</property>
-                <property name="position">0</property>
-              </packing>
-            </child>
             <child>
-              <object class="GtkBox" id="box1">
+              <object class="GtkBox" id="task_toolbar_box">
```

Comment:
This is a great idea!
However, there is a problem with the default padding property for child elements in GtkButtonBox (yes, I'm convinced the problem is there, I've spent a lot of time on this and I cannot find any other possible explanation) which causes the child buttons to be separated and not nicely aligned.

If you have an idea, I'll be glad to implement it, but for now, child-internal-pad-x is a property which I cannot implement and re-set to the value of 0, which would solve our problem.